server/monitor.ts

```diff
@@ -213,6 +217,7 @@ export const Monitor = new class {
 	 * Counts group chat creation. Returns true if too much.
 	 */
 	countGroupChat(ip: string) {
+		if (Config.noipchecks) return;
```

Comment:
Most of these should be `return false` rather than `return`. Look at the function signatures.